### PR TITLE
fix: change LicensesInitials to LicenseInitials

### PR DESCRIPTION
**Pull request by yamii**
@sinet/chaos-monkey please review

### CODE REVIEW

File: schemas/user/complete.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ module.exports = function ( keyArray ) {
 		'LastLogin'        : Joi.date().allow( null ).description( 'The date that the user last logged in' ),
 		'LastName'         : Joi.string().max( 64 ).description( 'The last name for the user' ),
 		'LicenseAccepted'  : Joi.date().allow( null ).description( 'The date that the user accepted the license' ),
-		'LicensesInitials' : Joi.string().allow( null ).max( 10 ).description( 'The initials the user used for accepting the license' ),
+		'LicenseInitials' : Joi.string().allow( null ).max( 10 ).description( 'The initials the user used for accepting the license' ),
 		'LoginCount'       : Joi.number().integer().positive().max( 5 ).allow( 0 ).description( 'The number of times the user has logged in' ),
```
Comment (sydcanem) [inline schemas/user/complete.js]: @yamii `:` seems not aligned on this new change.

Comment (openam) [reply schemas/user/complete.js]: lol should we add `eslint` to this?

Comment (yamii) [reply schemas/user/complete.js]: @openam @sydcanem it's fix now

**Review by openam — APPROVED** [review]

**Comment by openam** [comment]
@yamii will you add a `release: 1.1.3` commit.

**Comment by yamii** [comment]
@openam added release

**Pull request merged**